test/net/sourceforge/kolmafia/maximizer/MaximizerRegressionTest.java

```diff
@@ -109,6 +112,15 @@ public void keepsBonusBjornUnchanged() {
     assertEquals(25, modFor("Meat Drop"), 0.01);
   }
 
+  @Test
+  public void equipEmptyBjornNoSlot() {
```

Comment:
If we agree that this is a good test and the fact that is fails indicates an error to be fixed, then we should probably add the Disabled annotation here with a comment to that effect.